### DISCUSSION TITLE
'DBeaver Community Edition' is Apache license, is it available for commercial use?

### DISCUSSION THREAD

**Discussion by takumi1110** [dbeaver/dbeaver#10656] [Q&A] [answered]
I don't have any knowledge about license, so please let me know.

'DBeaver Community Edition' is Apache license, is it available for commercial use?

**Comment by serge-rider** [comment, marked as answer]
Yes, of course. 
There are no limitations.

**Reply by takumi1110** [reply]
Thank you for creating such a great app!

I will use it for work at the company! !!
